docs/howto/kubernetes/gardener/kubectl.md

```diff
@@ -48,26 +73,57 @@ kubectl config view
 
 You should see something like this:
 
-```yaml
-apiVersion: v1
-clusters:
-  - cluster:
-      certificate-authority-data: DATA+OMITTED
-      server: https://api.test-cluster.p40698.staging-k8s.{{gui_domain}}
-    name: shoot--p40698--test-cluster
-contexts:
-  - context:
-      cluster: shoot--p40698--test-cluster
-      user: shoot--p40698--test-cluster-token
-    name: shoot--p40698--test-cluster
-current-context: shoot--p40698--test-cluster
-kind: Config
-preferences: { }
-users:
-  - name: shoot--p40698--test-cluster-token
-    user:
-      token: REDACTED
```

Comment:
Let's please stick with one placeholder. I tend to use `REDACTED` because it's just one word. If you insist that `DATA+OMITTED` is better you can go with that too, but please pick one.

Reply:
I did not pick either of those; they are in the actual `kubectl config view` outputs.